### PR TITLE
Create killmonger.yml

### PR DESCRIPTION
**Pull request by MH03**
Proof https://www.youtube.com/watch?v=38lqH9D9BaU and https://www.youtube.com/watch?v=trVSNjg6bRM

### CODE REVIEW

**Comment by malonnnn** [comment]
missing timestamps

**Comment by MH03** [comment]
for strength  https://www.youtube.com/watch?v=38lqH9D9BaU 0:23

**Comment by MH03** [comment]
for resistance https://www.youtube.com/watch?v=38lqH9D9BaU 0:11

**Comment by MH03** [comment]
https://www.youtube.com/watch?v=38lqH9D9BaU 0:06 for jump

**Comment by MH03** [comment]
https://www.youtube.com/watch?v=cs0Lc541ekg 0:38 run

**Comment by MH03** [comment]
https://www.youtube.com/watch?v=cs0Lc541ekg 0:49 for jump again

File: CommandHelper/LocalPackages/commands/chars/cloak.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class: cloak
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/chars/cloak.yml]: You put the cloak stuff in with the kilmonger stuff, i think you need to redo killmonger, cuz im not sure how u would remove cloak from this PR

thhanks

**Pull request closed without merging**